doc/classes/ProjectSettings.xml

```diff
@@ -1531,7 +1536,12 @@
 			An override for [code]rendering/gles3/shaders/ubershader_cache_size_mb[/code], so a smaller maximum size can be configured for mobile platforms, where storage space is more limited.
 			[b]Note:[/b] This setting is only meaningful if [code]rendering/gles3/shaders/shader_compilation_mode[/code] is set to [code]Asynchronous + Cache[/code].
 		</member>
-		<member name="rendering/gles3/shaders/shader_compilation_mode" type="int" setter="" getter="" default="2">
+		<member name="rendering/gles3/shaders/shader_cache_size_mb.web" type="int" setter="" getter="" default="128">
+			An override for [code]rendering/gles3/shaders/ubershader_cache_size_mb[/code], so a smaller maximum size can be configured for web platforms, where storage space is more limited.
```

Comment:
Same here, outdated name in the `[code]` reference, but using `[member]` would flag this.